az-pipelines/README.md

```diff
@@ -0,0 +1,34 @@
+# GitHub Action for triggering Azure YAML Pipelines
+
```

Comment:
Should there be some explanation regarding the behavior depending on artifact type? @nishubansal

Reply:
Lets keep it simple for now and trust action logs for these, else we might have to explain too many concepts around pipelines. Will update if we hear feedback.